examples/tactics/HandleSmtGoal.fst

```diff
@@ -0,0 +1,24 @@
+module HandleSmtGoal
+
+open FStar.Tactics
+
+[@@ resolve_implicits; handle_smt_goals]
```

Comment:
Just for clarification, the `resolve_implicits` attribute in this file is not relevant, right?